indent/verilog_systemverilog.vim

```diff
@@ -65,16 +65,16 @@ if exists("*GetVerilogSystemVerilogIndent")
   finish
 endif
 
-if exists("g:verilog_disable_indent")
-  let s:verilog_disable_indent = split(g:verilog_disable_indent, ",")
-else
-  let s:verilog_disable_indent = []
-endif
-
 set cpo-=C
 
 function! GetVerilogSystemVerilogIndent()
 
+  if exists("g:verilog_disable_indent")
+    let s:verilog_disable_indent = split(g:verilog_disable_indent, ",")
+  else
+    let s:verilog_disable_indent = []
+  endif
+
```

Comment:
This doesn't actually need to get run here, and maybe inefficient to run it on every call to the indent script. Would it not be better to wrap any checks to `g:verilog_disable_indent` to run this first?